discoverable-client/src/main/java/com/ca/mfaas/client/api/FileController.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package com.ca.mfaas.client.api;
+
+import io.swagger.annotations.Api;
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.ServletContext;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+
+/**
+ * Version 1 of the controller that returns a zip file.
+ */
+@RestController
+@Api(tags = {"Other Operations"})
+public class FileController {
+
+    private final ServletContext servletContext;
+    private static final String DIRECTORY = "C:\\Users\\cz670745\\myRepos\\api-layer\\discoverable-client\\src\\main\\resources";
```

Comment:
Be careful with that ;-)

Reply:
Ooopss. Sorry for that :)

Reply:
Hi @cZikos & @JiraAichler

Thanks for adding test and reviewing, will join api-ml call tomm 08:00AM EST.

Added setting to prevent Gateway from decompressing application/gzip response
Didn’t realize application/zip response from Service was being uncompressed too.

FYI, Common case for us is all modern browser clients make request with 
Accept: gzip, deflate, br
And we added this feature to dataset-api and jobs-api recently to respond to these headers.

Reply:
Hello @NakulManchanda 
I added the compression as you suggested, that was my fault. I also added a relative path to resources.